src/components/RadioButtons/RadioButtons.js

```diff
@@ -1,63 +1,17 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import uuidv4 from 'uuid/v4'
+import useErrorMessage from '../../hooks/useErrorMessage.js'
 import useIncludes from '../../hooks/useIncludes.js'
 import useInvalid from '../../hooks/useInvalid.js'
 
 import { Body } from '../Type/Body.js'
 import { COLORS } from '../Colors.js'
 import styles from './RadioButtons.module.scss'
 
-// Note: RadioButtonGroup does not currently work (may have broken circa v0.9).
```

Comment:
I'm opting to completely remove the docs since they're from a previous version and I'd rather we rebuild these up thoughtfully as out of date docs will do more harm then good imo

Reply:
hard to disagree with